go/src/koding/kites/kloud/provider/google/bootstrap.json.tmpl

```diff
@@ -0,0 +1,52 @@
+{
+  "provider": {
+    "google": {
+      "credentials": "${var.google_credentials}",
+      "project": "${var.google_project}",
+      "region": "${var.google_region}"
+    }
+  },
+  "output": {
```

Comment:
The bootstrap template is expected to produce output variables - we need to output some variables here (the network id, firewall rule etc.) in order to set Bootstrap struct after bootstrapping.

The frontend calls bootstrap, when bootstrap struct is empty; if bootstrap struct is not updated after successful  bootstrap, then next time template is saved - the bootstrap is going to be called again.